tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -439,5 +440,12 @@ def test_gptq(self):
         GumbelLearnRateZeroNoTempLearnTest(self).run_test()
 
 
+    def test_qat(self):
+        """
+        This test checks the QAT feature.
+        """
+        QuantizationAwareTrainingTest(self).run_test()
```

Comment:
Consider adding tests for the following cases (for coverage):
- finalize=True
- weights_quantization_method=SYMMETRIC
- weights_quantization_method=UNIFORM

You already have the framework, duplicate this test call with the proper arguments

Reply:
I'm going to do it in my next task, once I fully implement QAT pytorch (currently it's only partial implementation)

Reply:
Can you add a "TODO" comment for this?